bool left_ir
bool center_ir
bool right_ir